### ISSUE TITLE
When sharing out to facebook the image pulled through is not always the first image on the post

### ISSUE THREAD

**Issue by jcalfee** [steemit/steemit.com#108]
for example
in this article https://steemit.com/money/@bravenewcoin/bravenewcoin-how-not-to-blockchain-a-look-at-onecoin

We just send the url to them.. they pull the page to get the title, description, and image url from our web page.  This page has https://s25.postimg.org/nreucme67/thumb.jpg as the image..  But FB consistently loads the first text image in the comment section.
![image](https://cloud.githubusercontent.com/assets/204121/17464954/1fd859c0-5cb8-11e6-8401-aa4b88e9fcaf.png)

**Comment by bmann** [comment]
The way to fix this is to set the meta tags of ```og:image``` and ```twitter:image``` as a defined image from the post.

Right now, Steemit has the same image for every single page that is shared. Ideally, the image for the post should be used to share.

In markdown, since we don't have a separate "post image", I would suggest that the first image is pulled. Since those are externally hosted, that might be tough.

**Comment by roadscape** [comment]
On the source of the linked post I see:

    <meta property="og:image" content="https://s25.postimg.org/nreucme67/thumb.jpg" data-reactid="13"/>
    <meta name="twitter:image" content="https://s25.postimg.org/nreucme67/thumb.jpg" data-reactid="19"/>

Why facebook continues to pull the random image from the comments section I'm not sure.

**Comment by jcalfee** [comment]
![image](https://cloud.githubusercontent.com/assets/204121/18129788/ede9b93e-6f51-11e6-9f7b-7ca80e6492b3.png)

**Comment by jcalfee** [comment]
The same thing happens when I upload my own family safe image to postimg.io.  I think it is safe to say FB does not work with this site.

**Comment by roadscape** [comment]
I have validated that this works as expected on Twitter, Pinterest, and Facebook.
https://cards-dev.twitter.com/validator
https://developers.pinterest.com/tools/url-debugger/
https://developers.facebook.com/tools/debug/

@jcalfee, that last image you tried might not have worked if it was too small. Most of these sites expect that the image is at least 120x120 or 200x200.

Closing, let me know if this issue pops up again..
